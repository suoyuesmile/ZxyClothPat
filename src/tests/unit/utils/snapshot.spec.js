import { genConfig, genConfig2 } from '@/utils/config'

describe('测试快照', () => {
  it('测试config', () => {
    expect(genConfig()).toMatchSnapshot({
      time: expect.any(Date)
    })
  })
  it('测试config2', () => {
    expect(genConfig2()).toMatchSnapshot()
  })
})
