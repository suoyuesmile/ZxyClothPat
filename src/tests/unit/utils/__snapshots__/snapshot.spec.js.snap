// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试快照 测试config 1`] = `
Object {
  "port": 80,
  "target": "www.test-api.com",
  "time": Any<Date>,
}
`;

exports[`测试快照 测试config2 1`] = `
Object {
  "port": 80,
  "target": "www.test-api.com",
  "time": "2019",
}
`;
